curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/6444ed90ff671201bf007af6.md

```diff
@@ -0,0 +1,223 @@
+---
+id: 6444ed90ff671201bf007af6
+title: Step 58
+challengeType: 0
+dashedName: step-58
+---
+
+# --description--
+
+While staying inside the function named `countingViews` you will create a if statement where it will check whether the argument `views` is more than or equal to 1000. A more than or equal to operator is represented by `>=`. If it is, then it will return the template literal saying `${thousands}k`. What it does is that it will return the number of thousands of views the post has. As an example, If it has 10000 views, it will return 10k.
```

Comment:
```suggestion
Within the `countingViews` function, create an `if` statement that checks whether the argument `views` is greater than or equal to 1000. A greater than or equal to operator is represented by `>=`. If it is, then it will return the template literal `${thousands}k`. This literal would represent 1000 as k, improving the readability. As an example, for 10000 views, it will return 10k.
```

Reply:
added